components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/IPublisherConfigServices.cs

```diff
@@ -51,6 +51,6 @@ public interface IPublisherConfigServices {
         /// <summary>
         /// Gets the diagnostic information for a specific endpoint
         /// </summary>
-        Task<PublishedNodesEntryModel> GetDiagnosticInfoAsync(PublishedNodesEntryModel request, CancellationToken ct = default);
+        Task<List<JobDiagnosticInfoModel>> GetDiagnosticInfoAsync(PublishedNodesEntryModel request, CancellationToken ct = default);
```

Comment:
`PublishedNodesEntryModel request` can be removed from input parameters, right ?

Reply:
yes